### PR TITLE
GREP: Informative warning message when removing a property on a dependency

### PR DESCRIPTION
**Pull request by habemuscode**
**Ticket**

https://oat-sa.atlassian.net/browse/ADF-794

### CODE REVIEW

File: views/js/uiForm.js [hunk 1]
```diff
@@ -36,7 +36,7 @@
     'ckeditor',
     'ui/ckeditor/ckConfigurator',
     'ui/datetime/picker',
-    'ui/dialog/confirmDelete',
+    'ui/dialog/confirm',
     'core/request',
```
Comment (404passionFruit) [inline views/js/uiForm.js:40]: Why are we changing the type of dialog? Aren't we breaking consistency on the platform?

Comment (habemuscode) [reply views/js/uiForm.js:40]: If you read the ticket you will see that this is the scope of the task, one of the ACs

**Review by clagosarias — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [ ]  New code is not subject to concurrency issues (if applicable)
- [ ]  The feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request's target is not master
- [x]  Pull request title and description are meaningful
- [x]  Commits are following conventional commits
- [x]  Kitchen links are not exposed in the description

File: views/js/uiForm.js [hunk 2]
```diff
@@ -506,23 +506,25 @@
...
-                        );
                     }
+
+                    const name = $groupNode.find('.property-heading-label')[0].innerText;
+                    const dependantPropName = dependencies.reduce((prev, next, index) => {
+                        const delimiter = index === dependencies.length - 1 ? '' : ', '
+                        return prev + `${next.label}${delimiter}`;
+                    }, '');
+
+                    let message = `<b>${name}</b>
```
Comment (hectoras) [inline views/js/uiForm.js:517]: nitpick: I think this can be a `const` as well

```suggestion
                    const message = `<b>${name}</b>
```

**Review by hectoras — APPROVED** [review]

File: views/js/uiForm.js (hunk 2 above)
Comment (habemuscode) [reply views/js/uiForm.js:517]: Good catch!

**Review by 404passionFruit — APPROVED** [review]

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**